src/Item/Item.router.ts

```diff
@@ -199,126 +182,88 @@ export class ItemRouter extends Router {
       )
     }
 
-    const [dbItems, remoteItems] = await Promise.all([
+    const [
+      dbItems,
+      remoteItems,
+      { dbTPItems, remoteTPItems },
+    ] = await Promise.all([
       Item.find<ItemAttributes>({ eth_address }),
       collectionAPI.fetchItemsByAuthorizedUser(eth_address),
+      this.itemService.getTPItemsByManager(eth_address),
     ])
 
-    const catalystItems = await peerAPI.fetchWearables(
-      remoteItems.map((item) => item.urn)
-    )
+    const [items, tpItems] = await Promise.all([
+      Bridge.consolidateItems(dbItems, remoteItems),
+      Bridge.consolidateTPItems(dbTPItems, remoteTPItems),
+    ])
 
-    return Bridge.consolidateItems(dbItems, remoteItems, catalystItems)
+    // TODO: add sorting (we're not breaking pagination)
```

Comment:
I'm not happy with the comment each time I read it either 🙃. What I'm trying to say is that doing: `list.concat(list2)` will not break pagination when we add it, but it will break any order we have beforehand and we need to add it later (after concatenating), cause if we don't it will have a different order each time

makes sense? should I clarify better on the comment?